### ISSUE TITLE
🐛 Unknown exceptions config property at dependencies

### ISSUE THREAD

**Issue by jamie-wearsafe** [rome/tools#1496]
![image](https://user-images.githubusercontent.com/52926764/118180212-4971dd80-b404-11eb-8af1-24434e72b7b8.png)

The configuration isn't working for adding license exceptions.

## Environment

```
$ rome rage --summary
Environment Variables
  Object {
    COLORTERM: "truecolor"
    LANG: "en_US.UTF-8"
    TERM: "xterm-256color"
    TERM_PROGRAM: "vscode"
  }

User Config
  unset

Rome Version
  10.0.4-beta

Node Version
  15.11.0

Platform
  darwin x64 20.4.0

Terminal Features
  Object {
    background: "unknown"
    colorDepth: 24
    columns: 142
    cursor: true
    hyperlinks: true
    isTTY: true
    unicode: true
  }

Client Flags
  Object {
    clientName: "cli"
    cwd: "/Users/jamienelson/wearsafe/test-build/github-action-jira-transition-manager"
    realCwd: "/Users/jamienelson/wearsafe/test-build/github-action-jira-transition-manager"
    silent: false
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 28_508_160
      pid: 47_966
      uptime: 0.249359019
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 28_508_160
        ownedBytes: 0
        ownedFileCount: 0
        pid: 47_966
        uptime: 0.249279336
      }
    ]
  }


```

## Steps To Reproduce

1. yarn rome config push dependencies.exceptions.invalidLicenses.APACHE "github-action-readme-generator@1.0.12"
2. 

## Expected Results

allows that invalid licence

**Comment by ematipico** [comment]
Yeah that's a problem of our documentation. It allowed to push documentation before actually releasing the feature. If you updated to version `10.0.7-nightly.2021.2.10`, that should be fixed.

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
